docs/api/commands/getallsessionstorage.mdx

```diff
@@ -37,15 +37,15 @@ Pass in an options object to change the default behavior of
 `cy.getAllSessionStorage()` yields an object where the keys are origins and the
 values are key-value pairs of `sessionStorage` data.
 
-For example, if `key1` is set to `value1` on `https://example.com` and `key2` is
-set to `value2` on `https://other.com`, `cy.getAllSessionStorage()` will yield:
+For example, if `key1` is set to `value1` on `https://origin1.com` and `key2` is
+set to `value2` on `https://origin2.com`, `cy.getAllSessionStorage()` will yield:
 
 ```js
 {
-  'https://example.com': {
+  'https://origin1.com': {
     key1: 'value1',
   },
-  'https://other.com': {
+  'https://origin2.com': {
```

Comment:
If we decide to go with localhost examples, these can be localhost on different ports.

Reply:
Ok, yah I wasn't sure if that would be explicit enough for the different origin examples to be useful for people.